more_itertools/more.py

```diff
@@ -2456,19 +2455,31 @@ def __getitem__(self, key):
         raise TypeError('islice_extended.__getitem__ argument must be a slice')
 
 
-def _islice_helper(it, s):
-    start = s.start
-    stop = s.stop
-    if s.step == 0:
+def _islice_helper(iterable, s):
+    start, stop, step = s.start, s.stop, s.step
+    if step == 0:
         raise ValueError('step argument must be a non-zero integer or None.')
-    step = s.step or 1
+    step = step or 1
 
     if step > 0:
-        start = 0 if (start is None) else start
```

Comment:
Ditto here; let's please not change things just to change them. The style enforcement for this repo is done by `black` and `flake8`.

Reply:
Sorry, I have changed this and what you mentioned above just because I edited the code nearby :) OK, I'll roll back these minor changes.